### ISSUE TITLE
String comparison marker points into emptiness after shortening the output

### ISSUE THREAD

**Issue by nohwnd** [pester/Pester#1263]
The `--------^` arrow is no longer needed after the output of string comparison was changed, but it is still present in string comparison, and it points to empty space. 

```
"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb" | Should -Be "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa0bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb"
Expected strings to be the same, but they were different.
Expected length: 122
Actual length:   121
Strings differ at index 81.
Expected: '...aaaaa0bbbb...'
But was:  '...aaaaabbbbb...'
----------------------------------------------------------------------------------------------------^
```

Please remove the arrow.

**Comment by t-mangoe** [comment]
Hello!
I tried this issue and edited files.
I want to push files I edited, but I don't have permission to push.
If you like, please give me permission.

**Comment by nohwnd** [comment]
You have to fork this repository, create a topic branch on your fork (git checkout -b “fix-arrow”), push it to your repository (git push) and then make a Pull Request. If you need more help, let me know :)

**Comment by t-mangoe** [comment]
Thank you,  nohwnd!!
I forked this repository and push my edited file. Then, I made Pull Request.

**Comment by nohwnd** [comment]
Fixed by #1264 

@t-mangoe if you put the `Fixes #number` as a code block in the PullRequest it does not work. I did not know either, but it seems logical :)
